boards.txt

```diff
@@ -10548,6 +10548,55 @@ Midatronics.menu.upload_method.dfuMethod.upload.protocol=2
 Midatronics.menu.upload_method.dfuMethod.upload.options=-g
 Midatronics.menu.upload_method.dfuMethod.upload.tool=stm32CubeProg
 
+
```

Comment:
I would not add new menu entry only for one board.
Please move it under generic F4 menu like Adafruit F405